CardScan/Classes/CreditCardUtils.swift

```diff
@@ -25,15 +41,12 @@ public struct CreditCardUtils {
     }
     
     public static func isValidBin(number: String) -> Bool {
```

Comment:
In the android code, I used the `determineCardNetwork` method to check for validity. This is more readable an extensible than checking for each possible card:
https://github.com/getbouncer/cardscan-android/blob/master/cardscan-base/src/main/java/com/getbouncer/cardscan/base/CreditCardUtils.kt#L125